v2/src/log_sink_console.c

```diff
@@ -1,18 +1,252 @@
+            MU_P_OR_NULL(file),
+            line,
+            MU_P_OR_NULL(func));
+        if (snprintf_result < 0)
+        {
+            /* Codes_SRS_LOG_SINK_CONSOLE_01_022: [ If any encoding error occurs during formatting of the line (i.e. if any printf class functions fails), log_sink_console.log_sink_log shall print Error formatting log line and return. ]*/
+            (void)printf(error_string);
+        }
+        else
+        {
+            bool error = false;
+
+            snprintf_result = MIN(snprintf_result, (int)buffer_size);
+            buffer += snprintf_result;
+            buffer_size -= snprintf_result;
+
+            /* Codes_SRS_LOG_SINK_CONSOLE_01_013: [ If log_context is non-NULL: ]*/
+            if (log_context != NULL)
+            {
+                /* Codes_SRS_LOG_SINK_CONSOLE_01_014: [ log_sink_console.log_sink_log shall call log_context_get_property_value_pair_count to obtain the count of properties to print. ]*/
+                size_t property_value_pair_count = log_context_get_property_value_pair_count(log_context);
+                /* Codes_SRS_LOG_SINK_CONSOLE_01_015: [ log_sink_console.log_sink_log shall call log_context_get_property_value_pairs to obtain the properties to print. ]*/
+                const LOG_CONTEXT_PROPERTY_VALUE_PAIR* property_value_pairs = log_context_get_property_value_pairs(log_context);
+                int log_n_properties_result =  log_n_properties(buffer, buffer_size, property_value_pairs, property_value_pair_count);
+                if (log_n_properties_result < 0)
+                {
+                    /* Codes_SRS_LOG_SINK_CONSOLE_01_022: [ If any encoding error occurs during formatting of the line (i.e. if any printf class functions fails), log_sink_console.log_sink_log shall print Error formatting log line and return. ]*/
+                    error = true;
+                }
+                else
+                {
+                    log_n_properties_result = MIN(log_n_properties_result, (int)buffer_size);
+                    buffer += log_n_properties_result;
+                    buffer_size -= log_n_properties_result;
+                }
+            }
+
+            if (!error)
+            {
+                if (buffer_size > 1)
+                {
+                    *buffer = ' ';
+                    buffer++;
+                    buffer_size--;
+
+                    va_list args;
+                    va_start(args, message_format);
+                    int vsnprintf_result = vsnprintf(buffer, buffer_size, message_format, args);
+                    if (vsnprintf_result < 0)
+                    {
+                        /* Codes_SRS_LOG_SINK_CONSOLE_01_022: [ If any encoding error occurs during formatting of the line (i.e. if any printf class functions fails), log_sink_console.log_sink_log shall print Error formatting log line and return. ]*/
+                        error = true;
+                    }
+                    else
+                    {
```

Comment:
We can remove the else part .

Reply:
Added a comment to make it clear that we don't expect to do anything on that branch, but keeping the branch anyway to show it's not missed.